samples/nextjs/src/pages/[[...path]].tsx

```diff
@@ -36,14 +39,28 @@ const SitecorePage = ({ notFound, layoutData, componentProps }: SitecorePageProp
 
 // This function gets called at build and export time to determine
 // pages for SSG ("paths", as tokenized array).
-export const getStaticPaths: GetStaticPaths = async () => {
+export const getStaticPaths: GetStaticPaths = async (context) => {
   // Fallback, along with revalidate in getStaticProps (below),
   // enables Incremental Static Regeneration. This allows us to
   // leave certain (or all) paths empty if desired and static pages
   // will be generated on request.
   // See https://nextjs.org/docs/basic-features/data-fetching#incremental-static-regeneration
   //
-  // Ultimately, this is where we'll also be able to request a "sitemap" from Sitecore.
+
+  if (process.env.BUILD_MODE === 'export') {
+    const ROOT_ITEM = `/sitecore/content/${config.jssAppName}/home`;
+    const paths = await graphQLSitemapService.fetchSitemap(
+      context.locales || [],
+      ROOT_ITEM,
+      nextConfig.i18n.defaultLocale
```

Comment:
Should be able to use `context.defaultLocale` here

Reply:
Don't need `defaultLocale` - we decided to use single language for Export. And we will not have language embedding